App/dApp.py

```diff
@@ -1,20 +1,22 @@
 # The web-application for our blockchain
```

Comment:
I saw a way to upload file to the dApp server in this tutorial

https://www.roytuts.com/python-flask-multiple-files-upload-example/

I will copy this file and run it with that and try to see how it works. Then I guess the other thing is setting the Mongo database